docs/user-guide/configuration/workflow.md

```diff
@@ -152,11 +154,44 @@ In the following example, this pipeline will start the `main` job after any pull
 jobs:
     main:
         image: node:6
-        requires: [~pr, ~commit, ~sd@456:publish]
+        requires:
+            - ~pr
+            - ~commit
+            - ~sd@456:publish
```

Comment:
I actually like this format less since it takes up more lines and makes the yaml longer, but to each their own, I guess.

Reply:
Changed it to be consistent since we are using `-` for the `steps`. Might be confusing to someone who is new to `yaml`. But I don't mind if you want to change back.

Reply:
Mm yeah let's change it back. I'll make `steps` more consistent later